README.md

```diff
@@ -90,7 +90,8 @@ Type: `String[]`<br>
 Default: `[]`
 
 ### require
-Require files/dir before executing features. If you apply a path don't use globbing as it is not supported by Cucumber. Instead just reference the directory where you step definitions are located (e.g. `/path/to/definitions` instead of `/path/to/definitions/*`).
+**Please note that globbing is not supported by Cucumber, paths must adhere to Node module resolution**
+Require files before executing features (e.g. `/path/to/module` or `/path/to/module-directory` where directory contains `index.js`).
```

Comment:
@christian-bromann Yeah that's the problem.

Feel free to close this pull request if you've got better documentation in mind 😊 

Personally, I'd prefer clarity that:

1. The "require all in directory" behaviour is now gone
2. The `require` config now follows Node module resolution
3. Globbing (e.g. `./path/*`) still isn't supported

Perhaps using an `index.js` file isn't widely known, but it's probably where I'd put all my `require()` or `import` calls so I can manage them using code not config.

Otherwise add an example showing how to explicitly include them all, e.g.

```js
require: [
    './path/to/steps/module1.js',
    './path/to/steps/module2.js'
    './path/to/steps/module3.js'
]
```

You've got *22k* monthly downloads via npmjs, likely others will want to know this too!

Thanks again for the hard work, WebdriverIO is a great framework.